content/en/flux/installation/bootstrap/github.md

```diff
@@ -0,0 +1,78 @@
+---
+title: GitHub
+linkTitle: GitHub
+description: "How to follow bootstrap procedure for Flux using GitHub"
+weight: 20
+---
+### GitHub and GitHub Enterprise
```

Comment:
```suggestion

```

Reply:
Make the same change on all guides please, we don't want to duplicate the title.